### PR TITLE
Initial Changes as per new json template.

### PR DESCRIPTION
**Pull request by msvinaykumar**
Signed-off-by: msvinaykumar <vinakuma@redhat.com>

Changes to accommodate new JSON template as per Jira https://github.com/kruize/autotune/pull/486

### CODE REVIEW

**Comment by chandrams** [comment]
@msvinaykumar - I have deployed latest autotune from mvp_demo ( docker.io/msvinaykumar/autotune_operator:th) and applied the A/B json from PR #486, I get a NPE in autotune log, could you please check.

```
2022-08-2909:28:28.783 ERROR [qtp238816832-33][CreateExperimentTrial.java(74)]-java.lang.NullPointerException: Cannot invoke "com.autotune.common.experiments.TrialInfo.getTrialNum()" because the return value of "com.autotune.common.experiments.ExperimentTrial.getTrialInfo()" is null
java.lang.NullPointerException: Cannot invoke "com.autotune.common.experiments.TrialInfo.getTrialNum()" because the return value of "com.autotune.common.experiments.ExperimentTrial.getTrialInfo()" is null
	at com.autotune.experimentManager.services.CreateExperimentTrial.lambda$doPost$0(CreateExperimentTrial.java:63)
	at java.base/java.util.Arrays$ArrayList.forEach(Unknown Source)
	at com.autotune.experimentManager.services.CreateExperimentTrial.doPost(CreateExperimentTrial.java:61)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:516)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:400)
```

File: src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java [hunk 1]
```diff
@@ -131,51 +131,44 @@ public void setExperimentTrial(ExperimentTrial experimentTrial) {
...
+                        this.experimentTrial.getResourceDetails().getNamespace(),
+                        this.experimentTrial.getResourceDetails().getDeploymentName(),
+                        trialDetails.getConfigData(),
+                        kubernetesServices
+                );
+                IntStream.rangeClosed(1, numberOFIterations).forEach(
+                        i -> {
+                            deploymentHandler.initiateDeploy();
+                            //Check if deployment is ready
+                            for (int j = 120; j > 0 && !deploymentHandler.isDeploymentReady(); j--) {
```
Comment (dinogun) [inline src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java:151]: Can we stick this magic number `120` into some global constant

File: src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java [hunk 2]
```diff
@@ -131,51 +131,44 @@ public void setExperimentTrial(ExperimentTrial experimentTrial) {
...
+                                try {
+                                    LOGGER.debug("Still deployment is not ready for ExpName {} trail No {}", this.experimentTrial.getExperimentName(), this.experimentTrial.getTrialInfo().getTrialNum());
+                                    Thread.sleep(1000);
```
Comment (dinogun) [inline src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java:154]: The sleep duration needs to be a global constant as well

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**